Mathlib/Algebra/Group/Basic.lean

```diff
@@ -496,7 +496,7 @@ theorem inv_mul' : (a * b)⁻¹ = a⁻¹ / b := by simp
 #align inv_mul' inv_mul'
 #align neg_add' neg_add'
 
-@[to_additive]
+@[to_additive (attr := simp)]
```

Comment:
If the simp is there in mathlib4 it gives linter errors; removing that simp makes everything check out.

Reply:
I think you need to have `simp` only on the multiplicative version?

Reply:
It doesn't make "everything check out" since the proof of my new lemma broke because it was missing. However, it's easy to work around it and potentially non trivial to solve, so I'm leaving a porting note and outsourcing the problem to #3309.

This doesn't have anything to do with additive vs multiplicative stuff. At this level, both worlds have the same simp lemmas.